test/functional/page_objects/dashboard_page_controls.ts

```diff
@@ -670,6 +688,10 @@ export class DashboardPageControls extends FtrService {
         `range-slider-control-${controlId} > rangeSlider__lowerBoundFieldNumber`,
         value
       );
+      await this.testSubjects.pressEnter(
+        // force the change without waiting for the debounce
+        `range-slider-control-${controlId} > rangeSlider__lowerBoundFieldNumber`
+      );
```

Comment:
This fixes flakiness in `x-pack/test/functional/apps/dashboard/group3/drilldowns/dashboard_to_dashboard_drilldown.ts` by forcing the range slider to submit its changes **right away** without a debounce.

Specifically, because the control group is now responsible for handling its own diffing, there is a slight delay in the Dashboard picking these up. In the case of the test, the selections made to the options list triggers unsaved changes (i.e. the save button is clickable), but we aren't waiting for the **range slider** selections to be actually applied before saving - so, there is a small chance that the range slider changes aren't actually saved into the dashboard input.